### PR TITLE
DBZ-1323 Initialize commitTime for out-of-order chunk

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1323

### CODE REVIEW

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java [hunk 1]
```diff
@@ -197,8 +194,7 @@ protected void outOfOrderChunk(final byte[] content) {
             LOGGER.warn("Got out of order chunk {}, recording artifical TX", new String(content));
         }
         txId = UNDEFINED_LONG;
-        timestamp = UNDEFINED_STRING;
-        commitTime = null;
+        commitTime = Instant.now();
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java]: @jpechane, could you elaborate a bit on that entire notion of "out of order chunk"? E.g. in form of a method comment. Setting the commit time to "now" doesn't reflect reality AFAICS, should we log a warning in that method perhaps? Or is it something which will happen regularly?

Comment (jpechane) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java]: @gunnarmorling This happens sometimes when the database is restarted after the crash, unfrotunately not reproducible.

Comment (gunnarmorling) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java]: Ok, can you add a method comment?

Comment (jpechane) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java]: @gunnarmorling javaDoc comment added

Comment (gunnarmorling) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/StreamingWal2JsonMessageDecoder.java]: Thx.

**Comment by gunnarmorling** [comment]
Applied; thx, @jpechane!

**Pull request merged**